docs/config/header.html

```diff
@@ -27,7 +27,26 @@
   <script type="text/javascript" src="$relpath^dynsections.js"></script>
   $treeview
   $search
-  $mathjax
+  <script type="text/x-mathjax-config">
+      window.MathJax = {
+      tex: {
+      tags: "ams",
+      inlineMath: [['^^','^^'], ['\\(', '\\)']],
+      packages: ['base', 'ams', 'color']
+      },
+      options: {
+      ignoreHtmlClass: 'tex2jax_ignore',
+      processHtmlClass: 'tex2jax_process'
```

Comment:
could you add a brief comment what these options are for?

Reply:
No, because I don't know what they do. This is just the default configuration that comes out of the MathJax 2->3 conversion script: https://mathjax.github.io/MathJax-demos-web/convert-configuration/convert-configuration.html

Reply:
Ok won‘t hurt to have these options